src/framework/components/anim/component-layer.js

```diff
@@ -299,10 +298,10 @@ class AnimComponentLayer {
     }
 
     /**
-     *  Returns the asset that is associated with the given state.
+     * Returns the asset that is associated with the given state.
      *
      * @param {string} stateName - The name of the state to get the asset for.
-     * @type {pc.Asset}
+     * @returns {pc.Asset} - The asset associated with the given state.
```

Comment:
Remove the `pc.`

Reply:
Also, remove the `-`

Reply:
Basically, browse some other `@returns` tags to see how they're done:

https://github.com/playcanvas/engine/blob/dev/src/asset/asset-registry.js#L311